.travis.yml

```diff
@@ -21,6 +21,7 @@ env:
 - ARGS="--resolver lts-7.24"
 - ARGS="--resolver lts-7.2"
 - ARGS="--resolver lts-12.20"
+- ARGS="--resolver lts-13.19"
```

Comment:
Why not `lts-13.30` which is the latest in lts-13? Is there some specific case to use GHC-8.6.4 and not 8.6.5?